src/status_im2/subs/communities.cljs

```diff
@@ -50,6 +50,13 @@
  (fn [communities [_ id]]
    (get-in communities [id :members])))
 
+(re-frame/reg-sub
+ :communities/current-community-members
+ :<- [:chats/current-chat]
```

Comment:
We have `:chats/current-chat` at several places, but it looks weird to me.
It's hard to follow its life cycle, for me it is better to pass an id instead of relying on some subscription. wdyt?
(probably ok in this case, as parent already relying on this subscription)

Reply:
im not sure i understand, what's the problem, could you elaborate ?

Reply:
> Explicit is better than implicit


There were several scenarios where we didn't pass values explicitly, and that leads to a bug
1. **Pinned message bug** - Even though we have access to chat-id, we were relying on `:chats/current-chat`, even when the current chat was closed, which caused a bug. [Code](https://github.com/status-im/status-mobile/pull/14885/files#diff-bdf749d4861990bf570c189355c0808d58a17f1bc536124fe182746d87007781) (Line 33)

3. **Famous Unknown community bug** - Instead of passing view-id explicitly (even though we were accessing screen-param in communities screen), we assumed `:view-id` already has `:community` as its value, as we are navigating. Due to a weird scenario, this `view-id` was `:home-stack` and lead to a bug. [Code](https://github.com/status-im/status-mobile/pull/14410/files#diff-1abdb083e4a14a5552793ebe14ced407956e46df7e4a0eb86f25e0f7291eaa87) (Line 248)

I just want to raise the point, we should try to pass values explicitly, especially in the cases we have access to that (like above 2), instead of relying on some third subscription.

Reply:
i mean, if you want a value for current chat it must be a current chat overwise there is a bug , so i still don't see the issue, if we pass chat-id but we want a current chat, and chat id is different from current chat something wrong, passing id is not always convenient, like, in this example, it was implemented incorrectly, so having current-chat subscription is convenient and more idiomatic i  would say